catalogs/hive-metastore-common/src/main/java/org/apache/gravitino/hive/converter/HiveTableConverter.java

```diff
@@ -88,6 +88,10 @@ public static Transform[] getPartitioning(Table table) {
           BUILDER extends BaseColumn.BaseColumnBuilder<BUILDER, COLUMN>, COLUMN extends BaseColumn>
       Column[] getColumns(Table table, BUILDER columnBuilder) {
     StorageDescriptor sd = table.getSd();
+    // Collect column names from sd.getCols() to check for duplicates
+    java.util.Set<String> columnNames =
+        sd.getCols().stream().map(f -> f.getName()).collect(java.util.stream.Collectors.toSet());
```

Comment:
plz remove the package name. And the map function can be simplified to `FieldSchema::getName`